catalog/core/catalog-core-commands/src/main/resources/bin/catalog-export

```diff
@@ -39,9 +39,11 @@ if [ -z "${d}" ] || [ -z "${z}" ] || [ -z "${k}" ]; then
     usage
 fi
 
+echo "Connecting to running DDF and exporting catalog to ${z}..."
 ./client "catalog:dump --include-content=${z} ${d}"
 if which jarsigner >/dev/null; then
+        echo "Running jarsigner on ${z}, which will prompt you to enter the keystore password to sign the zip file..."
         jarsigner -keystore ${k} ${d}/${z} ${a}
 else
     echo "Unable to find jarsigner, ensure that $JAVA_HOME and $JAVA_HOME/bin are set on the path."
```

Comment:
Should we do an 'exit 1' here?  This looks like processing will continue even though the jar isn't signed.